packages/bpk-component-bottom-sheet/src/BpkBottomSheet.tsx

```diff
@@ -51,20 +50,22 @@ export type Props = {
   isOpen: boolean;
 }
 
-const BpkBottomSheet = ({
-  actionText = '',
-  ariaLabelledby,
-  children,
-  closeLabel = '',
-  closeOnEscPressed = false,
-  closeOnScrimClick = false,
-  id,
-  isOpen,
-  onAction = () => null,
-  onClose,
-  title = '',
-  wide = false
-}: Props) => {
+export type Props = CommonProps & ({ ariaLabelledby: string } | { ariaLabel: string; });
+
+const BpkBottomSheet = (props: Props) => {
```

Comment:
why not keep it as it was previously and destructure it here? 🤔

Reply:
This is a limitation of the mutually exclusive aria props unfortunately. TS won't allow destructuring of those fields because it knows one will not be there so by keeping the `props` intact we can safely reference or pass them on (subject to performing checks for existence to keep TS happy).

In this component we just pass whichever we have by spreading the `props`

In BpkDialog wrapper we have an example of checking and using a specific one here:  https://github.com/Skyscanner/backpack/pull/3427/files#diff-8ae3b880e3772a84f85517514640da0930f295cffbdfd531fcf063a29ba84606R139

Reply:
I see, but we could also destructure the props here and spread the aria props rather than explicitly destructure them? Something like
```
const BpkBottomSheet = ({actionText = '', children, etc, ...ariaProps}) => {...}
```
?
In which case we'd also pass whichever we have

Reply:
Ah, yeah we could use that syntax to keep the destructure - will push an update 🚀